Changelog.md

```diff
@@ -13,6 +13,7 @@ Compiler Features:
  * NatSpec: Support NatSpec comments on events.
  * Yul Optimizer: Store knowledge about storage / memory after ``a := sload(x)`` / ``a := mload(x)``.
  * SMTChecker: Support external calls to unknown code.
+ * Wasm backend: Properly support both ``i32.drop`` and ``i64.drop``, and remove ``drop``.
```

Comment:
Is "wasm backend" the right component? "wasm yul dialect"? But it also fixes the binary transform, right?

Also, shouldn't this be in the bugfixes?

Reply:
It is called `Wasm backend` under bugfixes already, hence I went with the same name. Yeah could consider it also a bugfix.

Reply:
Moved to bugfixes.